doc/architectural/background-concepts.md

```diff
@@ -696,8 +696,8 @@ is to encode both the program and the set of states using an appropriate logic,
 mostly *propositional logic* and (fragments of) *first-order logic*.
 
 In the following, we will quickly discuss propositional logic, in combination
-with SAT solving, and show how to build a simple bounded model checker for
```

Comment:
Why remove this restriction?

Reply:
My original intention was to show bounded model checking on code that's essentially a finite-state automaton, but that's actually not very helpful. Should I add the restriction back?

Reply:
@peterschrammel Any further comments?